.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Please remove all files that should not be in the repository (vs, bin and etc.).